src/github.com/couchbase/sync_gateway/rest/bulk_api.go

```diff
@@ -303,6 +303,12 @@ func (h *handler) handleBulkGet() error {
 		return err
 	}
 
+	_, ok := body["docs"].([]interface{})
```

Comment:
What do you think about saving the results of the cast to []interface{} here, and avoiding the additional cast in the range statement below?